source/05-EL/04.ptx

```diff
@@ -12,7 +12,7 @@
     <subsection>
     <title>Activities</title>
 
-    <activity xml:id="activity-exp-graphs-intro">
+    <activity xml:id="activity-exp-graphs-intro-EL4">
```

Comment:
```suggestion
    <activity xml:id="activity-log-graphs-intro">
```

Reply:
To do this, I clicked on "Files changed", and then hit the + on line 15 like I was going to leave a comment.  Then the first button to the right of Preview is an "Add suggestion".

Go ahead and hit "commit suggestion" if you like this.